static/manifest.json

```
@@ -0,0 +1,16 @@
+{
+  "name": "TUM Mensa",
```

```suggestion
  "name": "Mensa-Homometer",
```

I think "Mensa-Homometer" would be a better name. "TUM Mensa" doesn't really reflect what this app is about?